src/api/node.js

```diff
@@ -10,26 +10,23 @@ define(function () {
   // Node or Selection?
   // TODO: write tests. unit or integration?
   Node.prototype.getAncestor = function (rootElement, nodeFilter) {
-    var isTopContainerElement = function (element) {
-      return rootElement === element;
-    };
     // TODO: should this happen here?
-    if (isTopContainerElement(this.node)) {
+    if (this.node === rootElement) {
       return;
     }
 
-    var currentNode = this.node.parentNode;
+    var currentNode = this.node;
 
     // If it's a `contenteditable` then it's likely going to be the Scribe
     // instance, so stop traversing there.
-    while (currentNode && ! isTopContainerElement(currentNode)) {
+    while ((currentNode = currentNode.parentNode) && currentNode !== rootElement) {
```

Comment:
100% agreed, what I intended is to get rid of the func var, so using `function isTopContainerElement()` instead of `var isTopContainerElement = function()` would do it as well. Do you prefer it that way?